unimus-backup-exporter.env

```diff
@@ -0,0 +1,22 @@
+#Unimus server address. Format http://[address]:[Port]
+unimus_server="http://foo.bar:8085"
+#Unimus API key
+unimus_api_key="insert api key here"
+#Unimus backup type. Valid options are latest or historical
+backup_type="latest"
+#Git username
+git_username="foo"
+#Git email
+git_email="foo@bar.org"
+#Git server protocal. Valid Options are ssh, http, https
+git_server_protocal="ssh"
+#Git server address. Format [address]
+git_server_address="192.168.4.5"
+#Git Server Port
+git_port="22"
+#Git Repo Address. Standared is [User][Repo Name]
```

Comment:
`for GitHub or GitLab standard is [User]/[Repo Name], for self-hosted repo [Repo Name]`